### PR TITLE
update HH extension to work after global made system updates

### PR DESCRIPTION
**Pull request by romankor**

### CODE REVIEW

**Comment by romankor** [comment]
Global made some updates

**Comment by mr-feek** [comment]
thanks! Could you just post a screenshot of it now, then I'll merge?

**Comment by nachishankar** [comment]
[screen shot 2018-09-28 at 4 59 32 pm 2](https://user-images.githubusercontent.com/5450895/46233256-efe63e80-c33f-11e8-8071-4014a2ba7981.png)
screen shot 2018-09-28 at 4 59 35 pm 2

**Comment by nachishankar** [comment]
screen shot 2018-09-28 at 4 59 32 pm 2

**Comment by nachishankar** [comment]
There isn't a dark.css file - in the manifest file it says css :'dark.css' also the 
Also, the js: file is named "gp_override.js" not "gp_override.bundle.js" - I don't know if that makes a difference.. 

manifest.json file:

js": [ "gp_override.bundle.js" ],
"css": ["dark.css"],

**Comment by nachishankar** [comment]
screen shot 2018-09-28 at 7 24 24 pm

**Comment by butl3r1** [comment]
I cannot get it to work either.  I tried downloading the updated files that romankor made but I get the same error nachishankar posted above.  I also tried looking at the changes made to the gp_override file and made them directly in my existing plugin but that did not fix it either :(

File: src/js/gp_override.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ import flatpickr from "flatpickr";
         }
     }, 1000);
     function isHandHistoryMenu() {
-      if ($('.k_c.skin__cell-header').text().includes('Hand History')){
+      if ($('.k_c.skin__cell-header').length > 0 && $('.k_c.skin__cell-header').text().includes('Hand History')){
```
Comment (clue-codes) [inline src/js/gp_override.js]: `if ($('.j_c.skin__cell-header').length > 0 && $('.j_c.skin__cell-header').text().includes('Hand History')){`

File: src/js/gp_override.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ import flatpickr from "flatpickr";
...
         return $('.skin__cell.q_c');
```
Comment (clue-codes) [inline src/js/gp_override.js]: `return $('.skin__cell.l_c');`

**Review by clue-codes — COMMENTED** [review]
Had to fiddle with these locally to get the box to show up

**Comment by clue-codes** [comment]
not leaving a PR because I haven't used Github in forever and I'm lazy.

**Comment by butl3r1** [comment]
The above worked for me, changing the skin cell headers as Xecese said.  Thank you so much!!!!

**Comment by CMaximus** [comment]
Are you able to clarify what needs to get edited? I have no programming skills but if I know where to cut and paste, I think I can muddle through.

**Comment by nachishankar** [comment]
but when I try to load the package I get the error - "dark.css" missing!

**Comment by nachishankar** [comment]
@gt2011 Can you send me your folder that you "load unpacked" onto chrome? - mine isn't working yet

**Comment by nachishankar** [comment]
Or just modified the gp_override.bundle.js file from the existing extension -with changes. Basically whatever you changed. I tried packing the file with the above changes. didn't work

**Comment by butl3r1** [comment]
I am not sure if I am supposed to upload any files here so I am not going to upload my unpacked version.

What I suggest doing is using your old plugin that previously worked...then navigate to the chrome extension folder where it is located and copy all the files to a different folder.  That fixes all the errors people posted about above.  Then go into the gp_override file in there and make all the changes that romankor posted at the top of this thread.  

After you do that then make the changes that Xecese wrote above.  In mine specifically I had to also change the "$" variable to "(0, _zepto2.default)" every place where there was a $ as well.  Like so:
  if ((0, _zepto2.default)('.j_c.skin__cell-header').length > 0 && (0, _zepto2.default)('.j_c.skin__cell-header').text().includes('Hand History')){
            return (0, _zepto2.default)('.skin__cell.l_c');

Hope that helps.

**Comment by nachishankar** [comment]
hey my email ID is nachishankar@gmail.com Can you just send me the override
file you have?
Nachi

**Comment by nachishankar** [comment]
> I am not sure if I am supposed to upload any files here so I am not going to upload my unpacked version.
> 
> What I suggest doing is using your old plugin that previously worked...then navigate to the chrome extension folder where it is located and copy all the files to a different folder. That fixes all the errors people posted about above. Then go into the gp_override file in there and make all the changes that romankor posted at the top of this thread.
> 
> After you do that then make the changes that Xecese wrote above. In mine specifically I had to also change the "$" variable to "(0, _zepto2.default)" every place where there was a $ as well. Like so:
> if ((0, _zepto2.default)('.j_c.skin__cell-header').length > 0 && (0, _zepto2.default)('.j_c.skin__cell-header').text().includes('Hand History')){
> return (0, _zepto2.default)('.skin__cell.l_c');
> 
> Hope that helps.

Hey my email ID is nachishankar@gmail.com - could you just send me your gp-override file?

**Comment by dchuntley** [comment]
Would be willing to send someone some btc if they can patch this, and have it easily fixable through a link.  My email is Dchuntley21@gmail.com.

**Comment by butl3r1** [comment]
Gmail blocks the zip file I can't send it like that.  I've given you everything you need to fix it above^^
1ab19c46d55026b505f0e763fb100440

**Comment by dchuntley** [comment]
Then navigate to the chrome extension folder where it is located and copy all the files to a different folder.
Sorry very computer illiterate here.  i open the extensions and then i can see the two files backgroud.html and options.html and i dont see a way to copy them.   Im sure im prob not even copying the right stuff but i could use help getting past this step.  
Thanks, D-boy

**Comment by butl3r1** [comment]
C:\Users\%USERNAME%\AppData\Local\Google\Chrome\User Data\Default\Extensions\<Extension ID>

For windows 10^ will be slightly different depending on what operating system you use.

**Comment by dchuntley** [comment]
I am using Windows 7.  This is where im stuck now lol.  im so sorry. 
stuck

**Comment by dchuntley** [comment]
D-Boy

**Comment by nachishankar** [comment]
screen shot 2018-09-30 at 2 59 10 pm 2

**Comment by nachishankar** [comment]
doest that look right? or do I have to do the replace $ with zepto thing too?

File: src/js/gp_override.js (hunk 2 above)
Comment (nachishankar) [reply src/js/gp_override.js]: Is  

> I am not sure if I am supposed to upload any files here so I am not going to upload my unpacked version.
> 
> What I suggest doing is using your old plugin that previously worked...then navigate to the chrome extension folder where it is located and copy all the files to a different folder. That fixes all the errors people posted about above. Then go into the gp_override file in there and make all the changes that romankor posted at the top of this thread.
> 
> After you do that then make the changes that Xecese wrote above. In mine specifically I had to also change the "$" variable to "(0, _zepto2.default)" every place where there was a $ as well. Like so:
> if ((0, _zepto2.default)('.j_c.skin__cell-header').length > 0 && (0, _zepto2.default)('.j_c.skin__cell-header').text().includes('Hand History')){
> return (0, _zepto2.default)('.skin__cell.l_c');
> 
> Hope that helps.

Hey, 
so in your code it is ('.j_c.skin__cell-header') but in the script it is ('.k_c.skin__cell-header') which one is it?  does it make a difference?
Also, is it "return $('.skin__cell.l_c');" ?  OR "return $('.skin__cell.q_c');"

**Comment by nachishankar** [comment]
Okay never mind!! Finally got it! :) :) Thanks all of you! :)

**Comment by nachishankar** [comment]
screen shot 2018-09-30 at 3 59 46 pm 2
For whoever needs - that got it to work!

**Comment by romankor** [comment]
OK , guys this PR is not going to work right now , seems global are updating the classes every day with something else , i am working on a workaround , keep you updated

**Comment by dchuntley** [comment]
Any updates would be very much appreciated.  Thank you to everyone who is taking their time to help get this fixed! :)

**Comment by mr-feek** [comment]
@romankor we could just insert our own div at the top of the page (above the header) if there is nothing else reliable.

**Comment by romankor** [comment]
@mr-feek , fixed it , should work now until further updates

**Comment by nachishankar** [comment]
> @mr-feek , fixed it , should work now until further updates

Hey romanov, the changes in the config.js file - where is that file? in the extension there are only 3 .js files I can see
background.bundle.js
gp_override.bundle.js
options.bundle.js
but I can't find any key words from that script in any of these files. :|

**Comment by romankor** [comment]
what are you trying todo ? did you build the extension ? 
Clone the branch, run `yarn dist` , a `build` folder will be created , you can load the extension from that folder in debug mode.

**Comment by nachishankar** [comment]
damn. I'm a noob. I just made all those changes in the existing gp_override_bundle.js file - It somehow still  works though hehe.

**Comment by mr-feek** [comment]
I need to update the readme to explain in more detail how to build the app - I'll get around to making a new release and updating the docs later this week. Thank you for the input everyone

**Comment by mikesomething** [comment]
Bump. 

+1 for new release

**Comment by dschulz1** [comment]
Will paypal someone if they can fix mine

**Comment by nachishankar** [comment]
Hey man, the fix is already there. ping me on skype - twokeywood. I'll share it with you

**Comment by mr-feek** [comment]
thanks @romankor !

**Comment by beeeks** [comment]
First I want to say thank you so much for everyone's hard work.

I have a computer science bachelors and I've spent 5+ fully dedicated hours trying to get this working again.  I kept thinking I almost had it and I'm not a fan of asking for help...but I feel like I'm missing something small that's preventing my success.

$50 in bitcoin if somebody can help me get this working.

After modifying gp_override.js ... when i try to load the chrome extension it says error could not load icon 'icon-34.png' for browser action.  could not load manifest.

I'm also confused about the run 'yarn dist'.  is this like a command line thing?  I forget how I built this all the first time many months ago.

Does one only need the chrome extension or is the converter source code needed too?  How do you 'combine' them?

replying here or emailing me at brian456github@gmail.com would be so greatly appreciated and rewarded.

Thanks!

snip
 

//edit - solved nachishankar shared his working code with me and woulnd't take my bitcoin.  Thanks!  I still don't understand why the repository here doesn't work.  I'm willing to put a little time in to figure out putting the working directory (extension) here if that's not stepping on anybodies toes.  feek that ok?

**Comment by mr-feek** [comment]
@beeeks 

I just updated the `Readme.md` file in this repository with step by step installation instructions. I just installed it on a fresh computer and the steps above worked for me properly.

If you are having trouble with a specific step, feel free to open a new issue here

**Comment by CMaximus** [comment]
Is anyone else having an issue with their updated chrome extension? Mine was working this morning, now it does not work.

**Comment by 0524432** [comment]
The fix from the site update a few weeks ago is also now broken for me after this morning. Any idea feeks? Thanks

**Pull request merged**